MultiPage-Clone/css/main.css

```diff
@@ -1,3 +1,528 @@
+  outline: none;
+  pointer-events: none;
+}
+
+.sidebar {
+  min-width: 100vw;
+  min-height: 100vh;
+  background-color: var(--White);
+  position: absolute;
+  z-index: 2;
+  padding-top: 90px;
+  display: none;
+}
+
+.hamburger-menu:has(input:checked) + .sidebar {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: var(--Light-beige);
+  opacity: 1;
+}
+.menu-nav {
+  padding-top: 30px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 23px;
+}
+
+.menu-nav .nav-links1 {
+  color: var(--Royal-blue);
+}
+
+.nav-links {
+  text-decoration: none;
+  color: var(--Black);
+  cursor: pointer;
+  font-family: "Poppins", sans-serif;
+  font-weight: 700;
+  font-size: 2.5em;
+  line-height: 1em;
+}
+
+.nav-links1 {
+  text-decoration: none;
+  color: var(--Black);
+  cursor: pointer;
+  font-family: "Poppins", sans-serif;
+  font-weight: 700;
+  font-size: 2.5em;
+  line-height: 1em;
+}
+.nav-links:hover {
+  color: var(--Royal-blue);
+  font-weight: 700;
+}
+
+.media-information {
+  margin: 0;
+}
+
+@media screen and (min-width: 540px) {
+  header {
+    height: 120px;
+  }
+  .header-content {
+    justify-content: space-between;
+    margin: 0px 50px;
+  }
+  .hamburger-menu {
+    display: none;
+  }
+  .for-header {
+    width: fit-content;
+    flex-direction: row;
+    align-items: center;
+  }
+  .for-title::before {
+    content: "/";
+    padding: 0px 15px 0px 10px;
+  }
+
+  .nav-links {
+    font-weight: 300;
+    position: relative;
+    font-size: 12px;
+  }
+
+  .hamburger-cont {
+    position: static;
+    width: fit-content;
+  }
+  .menu-nav {
+    flex-direction: row;
+    gap: calc(var(--gap) + 1vw);
+  }
+
+  .sidebar {
+    display: inline;
+    min-width: fit-content;
+    min-height: fit-content;
+    position: static;
+    padding-top: 0px;
+  }
+}
+
+/* main */
+main {
+  margin-top: 100px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: fit-content;
+}
+
+.background-index {
+  background: linear-gradient(
+    180deg,
+    var(--White) 270px,
+    var(--Dark-beige) 270px
+  );
+}
+
+.background-dark {
+  background: var(--Dark-beige);
+}
+
+.card-section {
+  background-color: var(--Light-beige);
+  width: 88vw;
+  height: fit-content;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+.shadow {
+  box-shadow: var(--shd, 0 1px 4px rgba(0, 0, 0, 0.6));
+}
+.personal-information {
+  height: fit-content;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 40px 0px;
+  gap: 30px;
+}
+.my-img {
+  width: 50%;
+  border-radius: 9999px;
+}
+
+.card {
+  max-width: 6ch;
+  text-align: center;
+}
+
+.blue-line {
+  background-color: var(--Royal-blue);
+  width: 20%;
+  height: 3px;
+  border-radius: 9999px;
+}
+.button {
+  display: flex;
+  flex-direction: row;
+  gap: 20px;
+  font-size: 28px;
+}
+
+.btn-card {
```

Comment:
![image](https://github.com/CodeYourFuture/Module-HTML-CSS/assets/118762745/ce89eb59-f283-4841-8bb8-25683dd63585)
 
 
 when i hover over the resume button, it changes color on the demo and yours doesnt!